te/tedp_docker/setup.sh

```diff
@@ -29,50 +29,60 @@
 # OF SUCH DAMAGE
 #**********************************************************************************************
 
-#!/bin/sh
-apt update  # To get the latest package lists
+#!/bin/bash
 cd /tmp/
+sed -i '/deb-src/s/^# //' /etc/apt/sources.list && apt update
```

Comment:
we wanted to install pre-depdencies of curl using `apt build-dep curl -y`. This needs deb-src to be uncommented in sources.list